backend/src/test/java/com/example/backend/service/RecipeIntegrationTest.java

```diff
@@ -26,29 +28,52 @@ class RecipeIntegrationTest {
     @Autowired
     RecipeRepository recipeRepository;
 
+    @BeforeEach
+    void addRecipeToRepo() {
+        recipeRepository.save(new Recipe("123", "Test", Category.ASIAN));
+    }
+
     @Test
     void getAllRecipesReturnAllRecipes() throws Exception {
         mockMvc.perform(get("/api/recipes"))
                 .andExpect(status().isOk())
-                .andExpect(content().json(
-                        """
-                                []
-                                """
-                ));
+                .andExpect(content().json("""
+                        [
+                        {
+                            "id": "123",
+                            "name": "Test",
+                            "category": "ASIAN"
+                        }
+                        ]
+                         """));
     }
 
     @DirtiesContext
     @Test
-    void addRecipe_ExpectRecipe() throws Exception {
-        mockMvc.perform(post("/api/recipes")
-                .contentType(MediaType.APPLICATION_JSON)
-                .content("""
+    void getRecipeById() throws Exception {
+        mockMvc.perform(get("/api/recipes/123"))
+                .andExpect(status().isOk())
+                .andExpect(content().json("""
                         {
-                            "id": "some title",
-                            "name": "some name",
+                            "id": "123",
+                            "name": "Test",
                             "category": "ASIAN"
                         }
-                        """))
+                         """));
+    }
+
+    @DirtiesContext
+    @Test
+    void addRecipe_ExpectRecipe() throws Exception {
+        mockMvc.perform(post("/api/recipes")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content("""
+                                {
+                                    "id": "some title",
```

Comment:
Eventuell mal Nachfragen ich meine die ID wird nicht überschreiben sondern bleibt bestehen

Reply:
würde in diesem Fall zeile 72    "id": "123",   so lassen